dev-docs/RFCs/v8.0/ui-module-rfc.md

```diff
@@ -0,0 +1,137 @@
+# UI Module RFC
+
+* **Author**: Xiaoji Chen
+* **Date**: December 2022
+* **Status**: **Pending Review**
+
+## Overview
+
+Visualization applications often require some chrome around the WebGL canvas to offer better user experience. While it is possible for every app to implement their own UI components that interact with deck.gl, it is no trivial effort to implement them well. 
+
+This RFC proposes to add a new module that packages common UI components that seamlessly integrate with deck.gl, including but not limited to:
+
+- Marker: HTML element pinned to a location in the world space
```

Comment:
Making a note to evaluate nebula.gl's HTMLOverlay, Maplibre's Marker implementation.

I could see this using a similar interface as an IconLayer, except the rendered result is a single HTML element.

Would a good expectation be one Marker per location, or could multiple markers be placed by one UI Control? There are obvious performance implications if there are too many markers, but it seems pretty typical to pin up to a couple dozen Markers.